exchange/bidder.go

```diff
@@ -521,6 +525,19 @@ func (bidder *bidderAdapter) doRequestImpl(ctx context.Context, req *adapters.Re
 	}
 	bidder.me.RecordOverheadTime(metrics.PreBidder, time.Since(pbsRequestStartTime))
 	httpCallStart := time.Now()
+
+	if tmaxAdjustments != nil && tmaxAdjustments.Enabled {
+		if deadline, ok := ctx.Deadline(); ok {
+			bidderTmax := deadline.Add(-time.Duration(tmaxAdjustments.BidderLatencyAdjustment * int(time.Millisecond)))
+			if int(bidderTmax.Sub(time.Now())) < tmaxAdjustments.UpstreamResponseDuration*int(time.Millisecond) {
```

Comment:
`BidderResponseMin`  represents the minimum duration to make round trip and get response from bidder server. Whereas `UpstreamResponseDuration` is duration PBS need to prepare auction endpoint response.

Therefor should use `BidderResponseMin`  and not `UpstreamResponseDuration`.


```
- if int(bidderTmax.Sub(time.Now())) < tmaxAdjustments.UpstreamResponseDuration*int(time.Millisecond)

+ if int(bidderTmax.Sub(time.Now())) < tmaxAdjustments.BidderResponseMin*int(time.Millisecond)
```

Reply:
https://github.com/prebid/prebid-server/pull/2750/commits/1c5438ac24b5c02613eaae5e77c52a3e08c0d8fd